bskyogcard/src/util/toShortUrl.ts

```diff
@@ -0,0 +1,16 @@
+export function toShortUrl(url: string): string {
+  try {
+    const urlp = new URL(url)
+    if (urlp.protocol !== 'http:' && urlp.protocol !== 'https:') {
+      return url
+    }
+    const path =
+      (urlp.pathname === '/' ? '' : urlp.pathname) + urlp.search + urlp.hash
+    if (path.length > 15) {
+      return urlp.host + path.slice(0, 13) + '...'
```

Comment:
A single ellipsis could be a reasonable alternative 👀 No biggie either way though, of course.